### ISSUE TITLE
memory: 241 Gb

### ISSUE THREAD

**Issue by c316** [wsNepal/wsAppInfo#1]
This is what the demo shows as memory... Is that the amount available or the amount being used, or is that number just really wrong? 

http://wsappinfo.meteor.com/

Also the user count is 16 and the graph shows 0 users.

**Comment by piyushcoader** [comment]
yeah thats the 241 GB is total memory of system , the count 16 means total 16 users in the app & the 0 graph user means there is no user online  :)
